src/Components/StatusPills/StatusPills.types.ts

```diff
@@ -1,6 +1,7 @@
 import { BehaviorModalMode } from '../../Models/Constants/Enums';
 import { DTwin } from '../../Models/Constants/Interfaces';
 import { IExpressionRangeVisual } from '../../Models/Types/Generated/3DScenesConfiguration-v1.0.0';
+import { VisualColorings } from '../BehaviorsModal/Internal/BehaviorSection/BehaviorVisualRuleSection';
 
 export type StatusPillsContainerWidth = 'compact' | 'wide';
 
```

Comment:
nit on the name of this file. It should be ColorPills. OR move your types to your own files so we can just delete the old component when you do your migration.